PURL-SPECIFICATION.rst

```diff
@@ -432,20 +432,18 @@ To parse a ``purl`` string in its components:
   - UTF-8-decode the ``version`` if needed in your programming language
   - This is the ``version``
 
-- Split the ``remainder`` once from right on '/'
+- Strip all leading and trailing '/' characters (e.g., '/', '//', '///' and
```

Comment:
Only **trailing** characters characters are relevant here to obtain `name` by splitting once from the right and be sure we don't end up on an empty segment (e.g. `namespace/name/`).

The leading characters will be removed by the subroutine that parses the `namespace`:

https://github.com/package-url/purl-spec/blob/8508b81e5958ed9566ee5ca698f4d188d9d1b571/PURL-SPECIFICATION.rst?plain=1#L445-L447